src/server/routes/api.js

```diff
@@ -138,6 +140,17 @@ module.exports = function(app) {
         });
     });
 
+  api.get('/admin/generateTeams/:eventAlias', requireAuth, roleAuth(roles.ROLE_ADMIN),
```

Comment:
change this to `/admin/exportTeams/:eventAlias` - we should use `generateTeams` for when we actually build the teams feature